### ISSUE TITLE
如何微调模型教程不明白的地方

### ISSUE THREAD

**Issue by futureflsl** [open-mmlab/mmclassification#259]
继承基础配置
修改分类头
修改数据集
这些都放在一个配置文件里面吗，还有这个配置文件放在哪个文件夹下面
_base_ = [
    '../_base_/models/resnet50.py',
    '../_base_/datasets/cifar10.py', '../_base_/default_runtime.py'
]
继承基础配置这个base使用是相对路径，表示上一级目录，修改分类头base是当前目录下面resnet50.py,这个教程初学者完全看不懂，建议改详细一点。
_base_ = ['./resnet50.py']
model = dict(
    pretrained=None,
    head=dict(
        type='LinearClsHead',
        num_classes=10,
        in_channels=2048,
        loss=dict(type='CrossEntropyLoss', loss_weight=1.0),
    ))
还有能给个最终配置文件我看看最终配置内容吗

**Comment by mzr1996** [comment]
你好，目前configs文件夹下，除了_base_文件夹以外的所有配置文件，都是最终配置内容。继承基础配置表示这个配置文件将会包含基础配置中的所有配置参数，并可以进行重载。
如果是初学者，可以参考B站的[教学视频](https://www.bilibili.com/video/BV1v64y1v73K)
文档有所不足的地方我们会不断改进，欢迎提出更多的建议~
